### ISSUE TITLE
Reproducing paper results using pre-trained models

### ISSUE THREAD

**Issue by hcl-work-dev** [rohanchandra30/Spectral-Trajectory-and-Behavior-Prediction#30]
Hello! I found your paper very insightful and interesting to read. Also thank you for creating this repo. I have some difficulty reproducing the results in the paper using the default resource and pre-trained models downloaded following your instructions and wonders if you could take a look at my questions.

When I tried to reproduce the Argo dataset results using the 1stS1new model with the default dataset downloaded from your Google drive, (s2 = False, TRAIN = False, EVAL = True) and got the following numbers:

ADE: 1.5727368501797476 FDE: 1.5565551090398055. 

Could you share with me your number of the above normalized Argo results using the 1stS1new model for a sanity check?

Since these are normalized results. Given what you presented in the paper is metric and the Argo dataset provided is also in meter units. I saved and multiplied back the.std() and x.std() normalization to the corresponding axis in the "acc"  variable in the "MSE" function. As a result, I got the following ADE, and FDE in meter units.

ADE: 849.5403021177667 FDE: 840.106767715443

I found these numbers really far away from what you reported in the paper and I wonder if you have any thoughts into what might have caused this. I have since tried Lyft and other datasets and see similar gaps.

The only difference I could think of are. 

1. I am evaluating your code using Conda in Windows, there are 2 Linux specific library that I replaced with Windows equivalents. One named "libgcc-ng" is not available on windows. My search tells the testing should no be affected by this library. 

2. Instead of using 2 Cuda GPUs, I used 1 and mapped all computation to map_location='cuda:0'.
 If your results are very different from what I mentioned above, any thoughts on what might be causing the issue?

Thanks in advance!

**Comment by rohanchandra30** [comment]
Hi, thank you ! :)

We recently made a lot of changes to the code because of which, the results are currently not reliable. I am going to update both the code and the paper results very soon. 

Please bear with us while we work to update the code/paper. Thanks!

**Comment by hcl-work-dev** [comment]
Got it, please kindly let me know when you finish the updates, thanks!
